test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -219,6 +220,92 @@ public async Task ParentInstanceId_Not_Assigned_In_Orchestrator(bool extendedSes
             }
         }
 
+        private static Mock<INameResolver> GetNameResolverMock((string Key, string Value)[] settings)
+        {
+            var mock = new Mock<INameResolver>();
+            foreach (var setting in settings)
+            {
+                mock.Setup(x => x.Resolve(setting.Key)).Returns(setting.Value);
+            }
+
+            return mock;
+        }
+
+        [Fact]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        public async Task WritesToConsole()
+        {
+            var expectedOutput = "MS_DURABLE_FUNCTION_EVENTS_LOGS";
+            string orchestratorName = nameof(TestOrchestrations.SayHelloInline);
+
+            // To capture console output in a StringWritter
+            using (StringWriter sw = new StringWriter())
+            {
+                // Set console to write to StringWritter
+                Console.SetOut(sw);
+                var nameResolver = new SimpleNameResolver(new Dictionary<string, string>()
+                {
+                    { "CONTAINER_NAME", "val1" },
+                    { "WEBSITE_STAMP_DEPLOYMENT_ID", "val3" },
+                    { "WEBSITE_HOME_STAMPNAME", "val4" },
+                });
+
+                // Run trivial orchestrator
+                using (var host = TestHelpers.GetJobHost(
+                    this.loggerProvider,
+                    nameResolver: nameResolver,
+                    testName: "CanWriteToConsole",
+                    enableExtendedSessions: false,
+                    storageProviderType: "azure_storage"))
+                {
+                    await host.StartAsync();
+                    var client = await host.StartOrchestratorAsync(orchestratorName, input: "World", this.output);
+                    var status = await client.WaitForCompletionAsync(this.output);
+                    await host.StopAsync();
+                }
+
+                this.output.WriteLine(sw.ToString());
+
+                // Ensure the console included some basic data
+                Assert.Contains(expectedOutput, sw.ToString());
+            }
+        }
+
+        [Fact]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        public async Task WritesToFile()
+        {
+            LinuxAppServiceLogger.LoggingPath = Path.Combine(Directory.GetCurrentDirectory(), "logfile.log");

```

Comment:
We may want to delete the path if it already exists, or add some suffix to our path, to ensure that we aren't just validating a previous run succeeded.

Reply:
I'll remove it before the test, if it exists. That's fair